BSB_LAN/BSB_LAN.ino

```diff
@@ -6564,49 +6566,62 @@ void loop() {
 
 
 #ifdef LOGGER
-#if defined(ESP32)
-  uint64_t freespace = 0;
-  freespace = SD.totalBytes() - SD.usedBytes();
-#else
-  uint32_t freespace = 0;
-  freespace = SD.vol()->freeClusterCount();
-#endif
-  if ((LoggingMode & CF_LOGMODE_SD_CARD) && freespace > MINIMUM_FREE_SPACE_ON_SD) {
+  if ((LoggingMode & (CF_LOGMODE_SD_CARD|CF_LOGMODE_UDP))) {
     if (((millis() - lastLogTime >= (log_interval * 1000)) && log_interval > 0) || log_now > 0) {
-//    SetDateTime(); // receive inital date/time from heating system
```

Comment:
Please don't remove commented code lines just because you don't think they're necessary. They may or may not be reminders for me to follow up on other functions.

Reply:
Understood, sorry.